src/Media/Svg/Size_Calculator.php

```diff
@@ -0,0 +1,88 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Libs\Media\Svg;
+
+class Size_Calculator {
+	private $original_width;
+	private $original_height;
+	private $original_aspect;
+	private $unreasonably_large_value = 9000;
```

Comment:
Ultimately, there's no great reason to go one way or another. I had considered making this a param we pass to the constructor or a filterable value, so it would need to be a property to handle that. But in the current implementation, there's no real difference.